lldb/include/lldb/Target/SwiftLanguageRuntime.h

```diff
@@ -2,7 +2,7 @@
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
```

Comment:
I think there are lots of changes that are just cosmetics, and can be separated. Like this.